source/vocab/files-packages-representations.ttl

```diff
@@ -0,0 +1,129 @@
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix prov: <http://www.w3.org/ns/prov#>
+prefix sdo: <http://schema.org/>
+prefix dc: <http://purl.org/dc/terms/>
+prefix premis3: <http://www.loc.gov/premis/rdf/v3/>
+#prefix edm: <http://www.europeana.eu/schemas/edm/>
+prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/>
+prefix : <https://id.kb.se/vocab/>
+
+##
+# PREMIS 3.0
+
+:Instance rdfs:subClassOf premis3:IntellectualEntity .
```

Comment:
I'm thinking if `Creation` instead should be `premis3:IntellectualEntity`?

Reply:
Makes sense (meaning `:Creation rdfs:subClassOf premis3:IntellectualEntity`?).

Reply:
I now see that my question was very vague. But yes exactly, as subClassOf :)